pkg/metrics/percentile/kll.go

```diff
@@ -77,7 +77,7 @@ func (s KLL) Quantile(q float64) float64 {
 		return s.Min
 	} else if q == 1 {
 		return s.Max
-	} else if s.Count <= int64(1/EPSILON)+1 {
+	} else if s.Count < int64(1/EPSILON) {
```

Comment:
Is this intended to be a change in behavior? i.e., the removing `=` and remove `+1` don't cancel out; both allow `s.Count` to be lower and still satisfy this condition

Reply:
Haha. I guess is the whole fix ;-) So, nvm.

Reply:
I could just have removed either the `=`, or the `+1` to fix the bug, but I wanted to have the same behavior as the other algos, which interpolate when the number of values is less than `1/epsilon`.